src/index.ts

```diff
@@ -1,13 +1,30 @@
 import { run } from 'f-promise';
 
-export function setup() {
-    const glob = global as any;
-    const it = glob.it;
-    if (it.wrapped) return;
-    glob.it = function (name: string, fn: () => void) {
-        return it(name, function (done: MochaDone) {
+
+function wrapWithRun(_fn: Function) {
+    return function (name: string, fn: () => void) {
+        if (!fn) {
+            return _fn(name);
+        }
+        return _fn(name, function (done: MochaDone) {
             run(() => fn()).then(done, done);
         });
     }
-    glob.it.wrapped = true;
+}
+export function setup() {
+    function patchFn(fnName: string, keep?: string[]) {
+        keep = keep || [];
```

Comment:
ok for renaming, but default value should stay like that to keep compliant with Node 4